packages/lesswrong/components/ea-forum/giving-portal/EAGivingPortalPage.tsx

```diff
@@ -49,6 +49,9 @@ const styles = (theme: ThemeType) => ({
     maxWidth: 1252,
     padding: 40,
     margin: "0 auto",
+    "@media screen and (max-width: 1000px)": {
```

Comment:
```suggestion
    [theme.breakpoints.down("md")]: {
```
The "md" breakpoint is 960px, is that close enough?